src/providers/WorkflowCore.Persistence.EntityFramework/Models/PersistedSubscription.cs

```diff
@@ -30,7 +30,7 @@ public class PersistedSubscription
 
         public string SubscriptionData { get; set; }
         
-        [MaxLength(200)]
+        [MaxLength(500)]
```

Comment:
We need to generate migrations for all SQL Server, Postgres and MySQL when we change schema